contracts/staking/fuse/FuseStaking.sol

```diff
@@ -312,7 +305,7 @@ contract FuseStaking is
 			earnings,
 			buyResult[2],
 			msg.sender,
-			keeperPartInGoodDollar
+			keeperPartInFuse
```

Comment:
do we need to change the event variable names?

Reply:
I think yeah, done that